### PR TITLE
update nightly test with propper segmented validation

### PR DESCRIPTION
**Pull request by jdnurme**
Update nightly test to properly validate. Tested full run manually prior to committing this change.

- [x] Tests pass
- [x] Appropriate changes to documentation are included in the PR

### CODE REVIEW

File: dataflux_core/performance_tests/list_and_download.py [hunk 1]
```diff
@@ -118,6 +119,7 @@ def run_download(self, config, list_result):
             raise AssertionError(
```
Comment (bernardhan33) [inline dataflux_core/performance_tests/list_and_download.py:119]: The check on the total download time and timeout doesn't have the "segmented" logic so the "download_timeout" is implicitly applied on a chunk of the download. Should we apply the similar logic here and check for the total download time in `test_list_and_download_one_shot`? Was that planned in a different PR?

**Review by bernardhan33 — APPROVED** [review]

File: dataflux_core/performance_tests/list_and_download.py (hunk 1 above)
Comment (jdnurme) [reply dataflux_core/performance_tests/list_and_download.py:119]: Yes, i'm going to ditch the timeout and gather metrics in the future so i didn't bother addressing.

Comment (jdnurme) [reply dataflux_core/performance_tests/list_and_download.py:119]: I'll update to a singular timeout in that cl

File: dataflux_core/performance_tests/list_and_download.py [hunk 2]
```diff
@@ -83,7 +83,7 @@ def run_list(self, config):
             )
         return list_result
 
-    def run_download(self, config, list_result):
+    def run_download(self, config, list_result, segmented=False):
```
Comment (MattIrv) [inline dataflux_core/performance_tests/list_and_download.py:86]: Should segmented be part of config?

**Review by MattIrv — APPROVED** [review]

File: dataflux_core/performance_tests/list_and_download.py (hunk 2 above)
Comment (jdnurme) [reply dataflux_core/performance_tests/list_and_download.py:86]: I realize i can just set segmented to be automatic for any expected data set larger than 50 gigs, so i'll update it to trigger off of the existing config.

**Pull request merged**